custom_components/myskoda/device_tracker.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, coordinator: MySkodaDataUpdateCoordinator, vin: str) -> None:
         )
         super().__init__(coordinator, vin)
 
-    def _positions(self) -> Positions:
+    def _positions(self) -> Positions | None:
```

Comment:
Can't be `None` since we're raising an exception if it is?

Reply:
Yeah im kind of struggling with this one: When the in-car user disables position, the capability is present, not available and the _positions() is None

Reply:
I think we should return `None` if it's existing but unavailable and just raise an exception if the capability isn't existing at all. But we're checking the later one before creating the device tracker at all.

So in summary I think returning `None` would be better than raising an exception.